README.md

```diff
@@ -48,17 +48,17 @@ Installation
 
 ### 1. From source
 
-The raw installation procedure is comprised of several steps that need to be
+The raw installation procedure is composed of several steps that need to be
 executed in order (fetching php sources, installing dev dependencies, building,
 creating the database, etc.). Except for the configuration step, the whole process
 is managed through composer scripts listed in the [composer.json](composer.json)
 file. For an installation from scratch, the commands would be:
 
     git clone http://github.com/claroline/Claroline
     cd Claroline
-    php scripts/configure.php
-    composer update
-     
+    php bin/configure
+    
+    composer update --no-dev -o
```

Comment:
Using the full option name instead of its shorthand might be clearer for documentation purposes, I believe shorthands are good when running commands by hand for faster typing:

```suggestion
    composer update --no-dev --optimize-autoloader
```